src/blenderbim/blenderbim/bim/module/pset/ui.py

```diff
@@ -67,18 +68,8 @@ def draw_psetqto_ui(context, pset_id, pset, props, layout, obj_type):
 
 
 def draw_psetqto_editable_ui(box, props, prop):
-    row = box.row(align=True)
-    if prop.data_type == "string":
-        row.prop(prop, "string_value", text=prop.name)
-    elif prop.data_type == "integer":
-        row.prop(prop, "int_value", text=prop.name)
-    elif prop.data_type == "float":
-        row.prop(prop, "float_value", text=prop.name)
-    elif prop.data_type == "boolean":
-        row.prop(prop, "bool_value", text=prop.name)
-    elif prop.data_type == "enum":
-        row.prop(prop, "enum_value", text=prop.name)
-    row.prop(prop, "is_null", icon="RADIOBUT_OFF" if prop.is_null else "RADIOBUT_ON", text="")
```

Comment:
This is an exception, where notice how there is no if statement checking whether or not the prop is_optional before displaying the is_null toggle. I've fixed this and will commit.

Reply:
Alright, sorry for that, I figured there would be some exceptions somewhere, I'm glad you took the time to review it. :) Don't hesitate to ping me if you stumble upon similar tasks you think I could handle.